doc/user-guide/combining.rst

```diff
@@ -43,7 +43,6 @@ new dimension by stacking lower dimensional arrays together:
 
 .. ipython:: python
 
-    da.sel(x="a")
```

Comment:
I don't think this was supposed to be here. It is not contributing to the combining docs.